fbc/v4.19/catalog/kueue-operator/catalog.json

```diff
@@ -114,16 +114,16 @@
     ],
     "relatedImages": [
         {
-            "name": "operand-image",
-            "image": "registry.redhat.io/kueue-tech-preview/kueue-rhel9@sha256:d0d6c34952e3d60be62fe7add33aa7ae2b0ac5c1bd2592e319f4cc28b2a2783e"
+            "name": "",
+            "image": "quay.io/redhat-user-workloads/kueue-operator-tenant/kueue-bundle-1-0@sha256:f2a9b5c73dada8854786fe08e8576d8fab4b109617138822b95b262bec0d2ae0"
```

Comment:
This seems a bad change.

Reply:
yea.  not expected.  this is coming from the bundle.  I installed the bundle directly and the pullspecs are using registry.redhat.io (in a mirrorset).  I don't know what is going on.